packages/api/src/controllers/stream.js

```diff
@@ -322,6 +323,26 @@ app.put('/:id/setactive', authMiddleware({}), async (req, res) => {
   res.end()
 })
 
+app.put('/:id/record', authMiddleware({}), async (req, res) => {
```

Comment:
For API consistency, would kind of prefer this to be a `app.patch("/:id")` with a body of `{"record": true}`. Should validate that too or currently we'll allow any valid JSON value to be saved to `req.body.record`